<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertiesPalette_ImageBase</class>
 <widget class="QWidget" name="PropertiesPalette_ImageBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="xposImgLabel">
       <property name="text">
        <string>X-Pos:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="ScrSpinBox" name="imageXOffsetSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="imageRotationLabel">
       <property name="text">
        <string>Rotation:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="ScrSpinBox" name="imageRotation">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="imagePageNumberLabel">
       <property name="text">
        <string>&amp;Page Number:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="ScrSpinBox" name="imageYOffsetSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="yposImgLabel">
       <property name="text">
        <string>Y-Pos:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ScrSpinBox" name="imagePageNumber">
       <property name="specialValueText">
        <string>Auto</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="alignmentGroupBox">
     <property name="title">
      <string>Alignment</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Horizontal</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Vertical</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="alignhorozintal">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Left</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Center</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Right</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="alignvertical">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Top</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Middle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bottom</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="scalingGroupBox">
     <property name="title">
      <string>Scaling</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="xscaleLabel">
        <property name="text">
         <string>X-Sc&amp;ale:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="ScrSpinBox" name="imageXScaleSpinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="yscaleLabel">
        <property name="text">
         <string>Y-Scal&amp;e:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="ScrSpinBox" name="imageYScaleSpinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="2" rowspan="2">
       <widget class="LinkButton" name="keepImageWHRatioButton">
        <property name="maximumSize">
         <size>
          <width>15</width>
          <height>32767</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="imgDPIXLabel">
        <property name="text">
         <string>Actual X-DPI:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="ScrSpinBox" name="imgDpiX"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="imgDPIYLabel">
        <property name="text">
         <string>Actual Y-DPI:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="ScrSpinBox" name="imgDpiY"/>
      </item>
      <item row="4" column="2" rowspan="2">
       <widget class="LinkButton" name="keepImageDPIRatioButton">
        <property name="maximumSize">
         <size>
          <width>15</width>
          <height>32767</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="freeScale">
        <property name="text">
         <string>Free</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QRadioButton" name="frameScale">
        <property name="text">
         <string>To Frame Size</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="cbProportional">
        <property name="text">
         <string>Proportional</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="colorMgmtGroup">
     <property name="title">
      <string>Color Management</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="1">
       <widget class="QComboBox" name="inputProfiles">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="renderIntentLabel">
        <property name="text">
         <string>Intent</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="renderIntent"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="inputProfLabel">
        <property name="text">
         <string>Input Profile</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="pdfCompressionGroup">
     <property name="title">
      <string>PDF Compression</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="compressionMethodLabel">
        <property name="text">
         <string>Method:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="compressionMethod">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="compressionQualityLabel">
        <property name="text">
         <string>Quality:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="compressionQuality">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="imgEffectsButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Image Effects</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="imgExtProperties">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Extended Properties</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScrSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>ui/scrspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>LinkButton</class>
   <extends>QToolButton</extends>
   <header>ui/linkbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>imagePageNumber</tabstop>
  <tabstop>imageXOffsetSpinBox</tabstop>
  <tabstop>imageYOffsetSpinBox</tabstop>
  <tabstop>imageRotation</tabstop>
  <tabstop>freeScale</tabstop>
  <tabstop>imageXScaleSpinBox</tabstop>
  <tabstop>imageYScaleSpinBox</tabstop>
  <tabstop>imgDpiX</tabstop>
  <tabstop>imgDpiY</tabstop>
  <tabstop>keepImageWHRatioButton</tabstop>
  <tabstop>keepImageDPIRatioButton</tabstop>
  <tabstop>inputProfiles</tabstop>
  <tabstop>renderIntent</tabstop>
  <tabstop>compressionMethod</tabstop>
  <tabstop>compressionQuality</tabstop>
  <tabstop>imgEffectsButton</tabstop>
  <tabstop>imgExtProperties</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
